kratos/includes/kratos_export_api.h

```diff
@@ -4,22 +4,21 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Carlos A. Roig Pina
 //
 
-#ifndef KRATOS_EXPORT_API_H
-#define KRATOS_EXPORT_API_H
+#pragma once
 
 #undef KRATOS_API_EXPORT
 #undef KRATOS_API_IMPORT
 #if _WIN32
-    #if defined(__MINGW32__) || defined(__MINGW64__)
+    #if defined(__MINGW32__) || defined(__MINGW64__) || defined(__INTEL_LLVM_COMPILER )

```

Comment:
I think it'd be easier the other way around: detect whether the compiler is MSVC, and set MSVC-specific API qualifiers if so. Every majro compiler (GCC, Clang, Intel) uses the standardized qualifiers.

As it stands right now, you're still missing the case when someone wants to use vanilla Clang on Windows.

Reply:
That may be the option..., the problem is that __INTEL_LLVM_COMPILER  is also detected as MSVC for some reason